MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -79,8 +79,7 @@ open class BaseInstructionsBannerView: UIControl {
     override open func prepareForInterfaceBuilder() {
         super.prepareForInterfaceBuilder()
         maneuverView.isStart = true
-        
-        primaryLabel.instruction = [VisualInstructionComponent(type: .destination, text: "Primary text label", imageURL: nil, maneuverType: .none, maneuverDirection: .none)]
+        primaryLabel.instruction = [VisualInstructionComponent(type: .text, text: "Primary text label", imageURL: nil, maneuverType: .none, maneuverDirection: .none, abbreviation: nil, abbreviationPriority: NSNotFound)]
```

Comment:
This is another observation as well. Should we consider wrapping our strings in `NSLocalizedString` to aid future translation work?

Reply:
Abbreviations will be localized server-side starting from this PR.